lib/rucio/common/dumper/consistency.py

```diff
@@ -166,7 +168,8 @@ def min3(*values):
     Minimum between the 3 values ignoring None
     '''
     values = [value for value in values if value is not None]
-    assert len(values) > 0
+    if len(values) <= 0:
+        raise ValueError("Not enough values to sort.")  # Need a better error message, but this function doesn't have a lot of info.
```

Comment:
I think this kind of message is fine considering the scope of the function - it's up to the caller to possibly propagate the exception with a more generic message, but in this context it makes sense to raise this kind of message. However, the name of the function indicates 3 values, even though this limit is not enforced anywhere in the function - that's a bit weird, but it's outside of the scope of this PR. I opened this issue for it https://github.com/rucio/rucio/issues/6990

len(values) can never be negative, so it should only be checked for 0.

```suggestion
    if len(values) == 0:
        raise ValueError("Input contains 0 non-null values.")
```